dataflow/src/ops/distinct.rs

```diff
@@ -0,0 +1,252 @@
+use std::collections::HashMap;
+
+use prelude::*;
+
+/// This will get Distinct elements from each group
+/// TODO
+
+#[derive(Clone, Serialize, Deserialize)]
+pub struct Distinct {
+    // Parent Node
+    src: IndexPair,
+
+    us: Option<IndexPair>,
+
+    group_by: Vec<usize>,
+
+    // A list of records that have been seen
+    seen: HashMap<Vec<DataType>, bool>,
```

Comment:
It doesn't look like this field is currently used anywhere? Was the idea to use this instead of the
```
let mut seen = HashMap::new();
```
inside `on_input`?

Reply:
It's also likely a good idea to document that these do not actually hold *all* records that have been seen, just the ones that have been seen in *this* packet. That is, they are cleared for every input.